modules/util/stixhelpers.py

```diff
@@ -214,13 +223,44 @@ def get_stix_memory_stores():
        that contains the memory stores for each domain.
     """
 
-    src = {}
+    # suppress InsecureRequestWarning: Unverified HTTPS request is being made
+    urllib3.disable_warnings(urllib3.exceptions.InsecureRequestWarning)
 
-    for domain in site_config.bundles:
-        src[domain] = stix2.MemoryStore()
-        src[domain].load_from_file(site_config.attack_path[domain])
+    ms = {}
+    srcs = []
 
-    return src
+    # Set proxy
+    proxy  = ""
+    if site_config.args.proxy:
+        proxy = site_config.args.proxy
+    proxyDict = { 
+        "http"  : proxy,
+        "https" : proxy
+    }
+
+    for domain in site_config.domains:
+
+        # Download json from http or https
+        if domain['location'].startswith("http"):
+            stix_json = requests.get(domain['location'], verify=False, proxies=proxyDict)
+            if stix_json.status_code == 200:
+                stix_json = stix_json.json()
+                ms[domain['name']] = stix2.MemoryStore(stix_data=stix_json['objects'])
+            elif stix_json.status_code == 404:
+                exit(f"\n{domain['location']} stix bundle was not found")
+            else:
+                exit(f"\n{domain['location']} stix bundle download was unsuccessful")
+        else:
+            if os.path.exists(domain['location']):
+                ms[domain['name']] = stix2.MemoryStore()
+                ms[domain['name']].load_from_file(domain['location'])
+            else:
+                exit(f"\n{domain['location']} local file does not exist. If you intended a URL, please include http:// or https://")
```

Comment:
👍  excellent error handling to tell the user what they did wrong in configuring the application!